addons/mail/static/src/core/web/messaging_menu.xml

```diff
@@ -102,6 +102,9 @@
                     </t>
                 </NotificationItem>
             </t>
+            <t t-if="store.channels.status === 'fetching'">
+                <span class="align-self-center p-1 o-visible-short-delay"><i class="fa fa-circle-o-notch fa-spin me-1"/> Loading…</span>
```

Comment:
I'm not sure what is the purpose of `o-visible-short-delay` but there's still a strange flicker when opening messaging menu when it loads data fast.

Reply:
changed to match size of "No conversation yet..."